vividus-plugin-web-app/src/main/java/org/vividus/selenium/WebDriverFactory.java

```diff
@@ -68,22 +69,24 @@ public WebDriver getWebDriver(DesiredCapabilities desiredCapabilities)
     @Override
     protected DesiredCapabilities updateDesiredCapabilities(DesiredCapabilities desiredCapabilities)
     {
-        WebDriverType webDriverType = WebDriverManager.detectType(desiredCapabilities);
-
-        Capabilities capabilities = desiredCapabilities;
-        if (webDriverType != null)
-        {
-            webDriverType.prepareCapabilities(desiredCapabilities);
-            configureProxy(webDriverType, desiredCapabilities);
-            if (webDriverType == WebDriverType.CHROME)
-            {
-                WebDriverConfiguration configuration = getWebDriverConfiguration(webDriverType, false);
-                ChromeOptions chromeOptions = new ChromeOptions();
-                chromeOptions.addArguments(configuration.getCommandLineArguments());
-                configuration.getExperimentalOptions().forEach(chromeOptions::setExperimentalOption);
-                capabilities = chromeOptions.merge(desiredCapabilities);
-            }
-        }
+        Capabilities capabilities = Stream.of(WebDriverType.values())
```

Comment:
WebDriverType.of(capabilities) ?

Reply:
I don't want to make `WebDriverType` dependent from `WebDriverManager` (see line below)

Reply:
As far as I understand, you have an enum and the name from caps, why it should be routed via WebDriverManager?

Reply:
`WebDriverType` will depend on `WebDriverManager` and `WebDriverManager` will depend on `WebDriverType`, so that's kind of cyclic dependency